app/views/dashboard/_subscription_status_widget.html.erb

```diff
@@ -3,7 +3,7 @@
 </h4>
 
 <% organizations = Organization.current.present? ? [Organization.current] : User.current.allowed_organizations %>
-<% subscriptions = organizations.collect { |org| org.redhat_provider.index_subscriptions }.flatten %>
+<% subscriptions = organizations.collect { |org| Katello::Pool.joins(:subscription).where("#{Katello::Subscription.table_name}.organization_id" => org.id) }.flatten %>
```

Comment:
Hooray for 1 line changes that have large impacts -- do you think this is worthy of a scope on the Pool or organizations model?